dear_petition/petition/models.py

```diff
@@ -332,3 +346,12 @@ class GeneratedPetition(TimeStampedModel):
     form_type = models.CharField(choices=FORM_TYPES, max_length=255)
     number_of_charges = models.IntegerField()
     batch_id = models.PositiveIntegerField()
```

Comment:
I'd suggest making all of the new fields read-only in the admin, so these values can't be modified.